Automattic/woocommerce-payments
# Resume sending E2E test failures to slack

Fixes #3555 

#### Changes proposed in this Pull Request
- Add the required config for using `e2e-environment` to resume sending E2E test failures to slack.
- Rename `jest-setup.js` to `jest.setup.js` and move it to `tests/e2e/config` for it to be picked up correctly.
- Use `useE2EJestConfig` for jest config to use the failure script that includes slack reporting.









#### Testing instructions



- Go to Actions tab ->  WC Pay E2E tests.
- Run the workflow for this branch `update/slack-e2e-reporting`.
- Notice failure messages would be sent to `#wcpay-bots` channel in slack.

Sample slack screenshot

<img width="639" alt="Screenshot 2021-12-24 at 1 12 28 PM" src="https://user-images.githubusercontent.com/68396823/147330555-f3027600-9fd6-401b-beda-1d4d5178a16b.png">


-------------------


**Post merge**
